talent-challenge-program/README.md

```diff
@@ -0,0 +1,42 @@
+# Talent Challenge Program
+
+Talent Challenge program is a platform that offers a structured remote mentoring program for **college students** successfully completing talent plan courses or having contributed to TiDB projects. Open source projects related to TiDB ecosystems will be selected together with professional mentors. Students **successfully completing talent plan courses or having contributed to TiDB projects** can apply for the projects, one at a time, and finish the projects with the help of the mentors in a given time. Mentees gain real-world experience by working closely with mentors. Students who successfully complete the projects will receive **stipends and better job opportunities**.
+
+### Current cycle
+
+Talent Challenge program season 1 is ongoing, check the timeline and projects available on [dedicated page](s1.md).
+
+## Communication
+
+Feel free to contact us in the **#talent-challenge** channel of the [tidbcommunity](https://slack.tidb.io/invite?team=tidb-community&channel=talent-challenge-&ref=github/talent-plan) slack workspace if any questions and suggestions about Talent Challenge program arise.
+
+## How to Participate
+
+You can participate in Talent Challenge program based on your role. Talent Challenge supports the following roles.
+
+### Project Maintainers/Mentors
+
+Project maintainers or mentors are requested to submit the ideas related to **TiDB Ecosystem** under the [Project Ideas](project-ideas.md) page using the [template](PROJECT_IDEA_TEMPLATE.md).
+
+PMC(Project Management Committee) of TiDB community will select the projects that will participate in the Talent Challenge program round and they will appear on the [Selected Projects](selected-projects.md) page after the selection.
+
+*Note:*
+
+- *Projects to be selected must have one mentor at least.*
+- *Mentors should be prepared to donate their time to help mentees with the project.*
+
+### Mentees
+
+*Note:*
+
+- Only **students who have completed talent plan courses or having contributed to TiDB projects** can apply for Talent Challenge program. 
```

Comment:
Can contributors apply for the project directly?

Reply:
the rest LGTM

Reply:
Contributors who are college students at school can apply for the projects